app/src/main/res/values/strings.xml

```diff
@@ -426,12 +426,12 @@
     <string name="termux">Termux</string>
     <string name="package_name_is_installed_successfully"><xliff:g id="app_name" example="App Manager">%1$s</xliff:g> installed</string>
     <string name="failed_to_install_package_name">Could not install <xliff:g id="app_name" example="App Manager">%1$s</xliff:g></string>
-    <string name="filter">Filter</string>
-    <string name="filter_user_apps">User apps</string>
-    <string name="filter_system_apps">System apps</string>
-    <string name="filter_apps_with_rules">Apps with rules</string>
+    <string name="filter">Filter apps</string>
```

Comment:
Not completely sure about these. May fail in some languages.
Pretty sure this convention of full naming exists other places too, (now inconsistent) so want to ask what you think.